public/app/ui/pages/home/HomePage.less

```diff
@@ -0,0 +1,21 @@
+article#home {
```

Comment:
These are your styles, moved from where you had it in `tweaks.less`, and I've just used a more consistent wrapper here, i.e. the outermost container element in the relevant template.